app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,5 @@
 <resources>
     <string name="app_name">JavaBaseProject</string>
+    <string name="textView_mainActivity">text</string>
+    <string name="textButton_mainActivity">boton</string>
```

Comment:
the name should be text_button_main_activity

Reply:
done